packages/shared/src/components/cards/WelcomePostCardHeader.tsx

```diff
@@ -1,32 +1,44 @@
 import React, { ReactElement } from 'react';
+import classNames from 'classnames';
 import { Post } from '../../graphql/posts';
 import { ProfilePicture } from '../ProfilePicture';
 import SourceButton from './SourceButton';
 import PostMetadata from './PostMetadata';
 
-type WelcomePostCardHeaderProps = Pick<Post, 'author' | 'source' | 'createdAt'>;
+type WelcomePostCardHeaderProps = Pick<
+  Post,
+  'author' | 'source' | 'createdAt'
+> & { enableSourceHeader?: boolean };
 
 export const WelcomePostCardHeader = ({
   author,
   createdAt,
   source,
+  enableSourceHeader,
```

Comment:
I would default it to `enableSourceHeader = false` just to keep it boolean always.